Jenkinsfile_CNP

```diff
@@ -61,16 +61,16 @@ withPipeline("java", product, component) {
   }
 
 //  env.IDAM_API_URL = 'https://idam-api.platform.hmcts.net'
-//  env.IDAM_S2S_AUTH_URL = 'http://rpe-service-auth-provider-prod.service.core-compute-prod.internal'
-//  env.CORE_CASE_DATA_API_URL = 'http://ccd-data-store-api-prod.service.core-compute-prod.internal'
-//  env.CLAIM_STORE_DB_HOST = 'prod-data-lb.moneyclaim.reform.hmcts.net'
-//  env.CLAIM_STORE_DB_PORT = '5432'
-//  env.CLAIM_STORE_DB_NAME = 'claimstore'
-//  env.CLAIM_STORE_DB_USERNAME = 'claimstore'
-//
-//  after('smoketest:prod') {
-//    sh './gradlew :ccd-claim-migration:assemble migrateClaims'
-//  }
+  env.IDAM_S2S_AUTH_URL = 'http://rpe-service-auth-provider-aat.service.core-compute-aat.internal'
+  env.CORE_CASE_DATA_API_URL = 'http://ccd-data-store-api-aat.service.core-compute-aat.internal'
+  env.CLAIM_STORE_DB_HOST = 'preprod-data-lb.moneyclaim.reform.hmcts.net'
+  env.CLAIM_STORE_DB_PORT = '5432'
+  env.CLAIM_STORE_DB_NAME = 'claimstore'
+  env.CLAIM_STORE_DB_USERNAME = 'claimstore'
+
+  after('smoketest:aat') {
+    sh './gradlew :ccd-claim-migration:assemble migrateClaims -Djava.util.concurrent.ForkJoinPool.common.parallelism=10'
```

Comment:
For 100 claims it should be fine. It will depend on how many records we want to test in next dry run. 
If we are choosing 1000 as suggested by Nick, then bump this to 20 or 25.

Reply:
we are running on eat with full payload, on prod can try with 1000